package.json

```diff
@@ -1,49 +1,42 @@
 {
-  "name": "xlsx-stream",
+  "name": "@brandwatch/xlsx-stream",
   "description": "Creates SpreadsheetML (.xlsx) files in sequence with streaming interface.",
-  "version": "0.1.2",
-  "homepage": "https://github.com/nunukim/node-xlsx-stream",
+  "version": "0.2.0",
+  "homepage": "https://github.com/BrandwatchLtd/node-xlsx-stream",
   "author": {
```

Comment:
Do we need to update the author info too? Seems out of date since the urls redirect oddly and doesn't apply to this codebase. Carried over from initial fork maybe?